packages/grafana-ui/src/saga-themes/code/themes/dark.ts

```diff
@@ -0,0 +1,67 @@
+export default {
+  color: {
+    background: {
+      ui: {
+        canvas: 'hsl(220 11.8% 7.2%)',
```

Comment:
Shouldn't these use the colors from the `colors` file, i.e., token, and not the actual values? Also wonder what does `ui` and `system` stand for?

Reply:
About the first question, the files that would be kept in sync are [these](https://github.com/grafana/grafana/pull/81977/files#diff-9d7e3a104e4389b993ec7f5be4c67a7f0db7e26752848fa99ab00ae600de28acR5) and they point at other tokens. The things inside `saga-themes/code/` are what is generated in this POC, we can add a comment to the top of each of these saying they are generated and probably change the name to be more clear like `dark.gen.ts` as well.

Regarding the `ui` and `system` additions, that would be a question for @Aires-Grafana

Reply:
Thanks for tagging me @JoaoSilvaGrafana .
the naming stuff about `ui` and `system` was an exploration to help differentiate the multiple kinds of background needs by categorising them:
`ui` was referencing the generic backgrounds to apply to the UI;
`system` was referencing the specific backgrounds to be used for system-related feedback.. like negative or positive feedback, warnings and informational components.